packages/components/Button/src/CompoundButton/__snapshots__/CompoundButton.test.tsx.snap

```diff
@@ -48,7 +48,7 @@ exports[`CompoundButton default 1`] = `
       numberOfLines={0}
       style={
         Object {
-          "color": "#323130",
+          "color": "#ffffff",
```

Comment:
Hm, this change doesn't seem correct

Reply:
This is intended, button does not have a default view on Android and iOS and this PR maps default to 'primary', this change is coming as a result of that.

Reply:
Ah okay makes sense. Thanks.